osgpkitools/incommon_request.py

```diff
@@ -60,172 +62,97 @@
-           (-H hostname | -f hostfile) [-a altnames] [-d write_directory]
-       %prog [--debug] -u username -k pkey -c cert -T
-       %prog -h
-       %prog --version'''
-    parser = OptionParser(usage, version=utils.VERSION_NUMBER)
-    group = OptionGroup(parser, 'Hostname Options',
-                        '''Use either of these options.
-Specify hostname as a single hostname using -H/--hostname
-or specify from a file using -f/--hostfile.''')
-    group.add_option(
-        '-H',
-        '--hostname',
-        action='store',
-        dest='hostname',
-        help='Specify the hostname or service/hostname for which you want to request ' + \
-        'the certificate for. If specified, -f/--hostfile will be ignored',
-        metavar='HOSTNAME',
-        default=None,
-        )
-    group.add_option(
-        '-f',
-        '--hostfile',
-        action='store',
-        dest='hostfile',
-        help='Filename with one host (hostname or service/hostname and its optional, ' + \
-        'alternative hostnames, separated by spaces) per line',
-        metavar='HOSTFILE',
-        default=None,
-        )
-    parser.add_option(
-        '-d',
-        '--directory',
-        action='store',
-        dest='write_directory',
-        help="Write the output files to this directory",
-        default='.'
-        )
-    parser.add_option(
-        '-k',
-        '--pkey',
-        action='store',
-        dest='userprivkey',
-        help="Specify requestor's private key (PEM Format)",
-        metavar='PKEY',
-        default=None
-    )
-    parser.add_option(
-        '-c',
-        '--cert',
-        action='store',
-        dest='usercert',
-        help="Specify requestor's user certificate (PEM Format)",
-        metavar='CERT',
-        default=None
-    )
-    parser.add_option(
-        '-a',
-        '--altname',
-        action='append',
-        dest='alt_names',
-        help='Specify an alternative hostname for CSR (FQDN). May be used more than ' + \
-             'once and if specified, -f/--hostfile will be ignored',
-        metavar='HOSTNAME',
-        default=[]
-    )
-    parser.add_option(
-        '-u',
-        '--username',
-        action='store',
-        dest='login',
-        help='Provide the InCommon username (login).',
-        metavar='LOGIN',
-        default=[]
-    )
-    parser.add_option(
-        '-T',
-        '--test',
-        action='store_true',
-        dest='test',
-        help='Test connection to InCommon API. Useful to test authentication credentials',
-        default=False
-    )
-    parser.add_option(
-        '',
-        '--debug',
-        dest='debug',
-        help="Write debug output to stdout",
-        action='store_true',
-        default=False
-    )
+    '''%(prog)s [--debug] -u username -k pkey -c cert \\
+           (-H hostname | -F hostfile) [-a altnames] [-d write_directory]
+       %(prog)s [--debug] -u username -k pkey -c cert -t
```

Comment:
Usage of -t changed to -T, that was deliberate? 

If you change usage message, don't forget to update README.md to match.

Reply:
Yes. I also changed -f to -F.
Changed README.md.

Reply:
I'm changing back -t\--test instead of -T

Reply:
I don't see README.md in the commit.

Reply:
My bad. The commit did not pick it up.